src/Workspaces/CSharp/Portable/Formatting/Engine/Trivia/TriviaRewriter.cs

```diff
@@ -195,22 +195,18 @@ public override SyntaxToken VisitToken(SyntaxToken token)
             // get token span
 
             // check whether we have trivia info belongs to this token
-            var leadingTrivia = token.LeadingTrivia;
-            var trailingTrivia = token.TrailingTrivia;
 
-            if (_trailingTriviaMap.ContainsKey(token))
+            if (_trailingTriviaMap.TryGetValue(token, out var trailingTrivia))
             {
                 // okay, we have this situation
                 // token|trivia
-                trailingTrivia = _trailingTriviaMap[token];
                 hasChanges = true;
             }
 
-            if (_leadingTriviaMap.ContainsKey(token))
+            if (_leadingTriviaMap.TryGetValue(token, out var leadingTrivia))
             {
                 // okay, we have this situation
                 // trivia|token
-                leadingTrivia = _leadingTriviaMap[token];
                 hasChanges = true;
             }
```

Comment:
Well, I added a second commit, but I'm not sure how to include it in this pull request?